packages/Autocompletion.package/ECController.class/instance/handleMenuShortcuts..st

```diff
@@ -0,0 +1,81 @@
+keyboard
+handleMenuShortcuts: aKeyboardEvent
+
+	| theEditor keyCharacter controlKeyPressed commandKeyPressed |
+	theEditor := self editor.
+	keyCharacter := aKeyboardEvent keyCharacter.
+	controlKeyPressed := aKeyboardEvent controlKeyPressed.
+	commandKeyPressed := aKeyboardEvent commandKeyPressed.
+	
+	menuMorph notifyOfInteraction.
+	"Home"
+	(keyCharacter = Character home or: [keyCharacter = Character arrowUp and: commandKeyPressed] "Mac compatiblity")
+			ifTrue: [
+				^ menuMorph home
```

Comment:
Just an idea:
We could refactor this:
``` smalltalk
^ menuMorph home
       ifTrue: [true]
       ifFalse: [self closeMenu. false]
```
to: 
``` smalltalk
^ menuMorph home
       ifFalse: [self closeMenu];
       yourself
```
Not quite sure which is better?